Source/WebCore/mathml/MathMLTokenElement.cpp

```diff
@@ -74,10 +74,10 @@ RenderPtr<RenderElement> MathMLTokenElement::createElementRenderer(RenderStyle&&
     return createRenderer<RenderMathMLToken>(*this, WTFMove(style));
 }
 
-bool MathMLTokenElement::childShouldCreateRenderer(const Node& child) const
+bool MathMLTokenElement::childShouldCreateRenderer(const Node&) const
 {
     // The HTML specification defines <mi>, <mo>, <mn>, <ms> and <mtext> as insertion points.
-    return isPhrasingContent(child) && StyledElement::childShouldCreateRenderer(child);
```

Comment:
Similarly can we just defer to the parent of MathMLElement here:   `StyledElement::childShouldCreateRenderer(child);`